src/start/start.S

```diff
@@ -209,10 +209,7 @@ terminate_hlt:
 GLOBAL(bsp_guard_page)
 	.fill	512, 8, 0
 bsp_stack_start:
-	.fill	512, 8, 0
-	.fill	512, 8, 0
-	.fill	512, 8, 0
-	.fill	512, 8, 0
+	.fill	8192, 8, 0
```

Comment:
This results in a stack that is 64K in size 8192 * 8 = 65,536. The original stack was 16,384 in size, not 4,096. Can you try with half that and see how it goes?

Also, can you increase the size of the AP stacks, too. An attestation request can come in on any AP.

Reply:
@tlendacky I increased the AP stacks to 9 pages (8 + 1 guard page) and changed the bsp to `.fill 4096, 8, 0`, which gives us 8 stack pages also for the bsp. However, I still see the same problem.

Enabling `FEATURES=verbose`, I see that it crashes just after the SNP_GUEST_REQUEST request is encrypted. Later I can add some more prints to see where exactly it's crashing.

```
Launching VM ...
  /tmp/cmdline.3286670
char device redirected to /dev/pts/25 (label compat_monitor0)
> All 17 firmware config files read.
> Starting SMP for 0 APs:
SecCoreStartupWithStack(0xFFFCC000, 0x820000)
INFO: SNP_GUEST_REQUEST message encrypted (96 bytes)
KVM: entry failed, hardware error 0xffffffff
EAX=0000000a EBX=00000000 ECX=00000000 EDX=00000000
ESI=00000000 EDI=00000000 EBP=00000000 ESP=00000000
EIP=00000000 EFL=00000002 [-------] CPL=0 II=0 A20=1 SMM=0 HLT=0
ES =0000 00000000 00000000 00000000
CS =0000 00000000 00000000 00000000
SS =0000 00000000 00000000 00000000
DS =0000 00000000 00000000 00000000
FS =0000 00000000 00000000 00000000
GS =0000 00000000 00000000 00000000
LDT=0000 00000000 00000000 00000000
TR =0000 00000000 00000000 00000000
GDT=     00000000 00000000
IDT=     00000000 00000000
CR0=80000033 CR2=00000000 CR3=00000000 CR4=00000668
DR0=0000000000000000 DR1=0000000000000000 DR2=0000000000000000 DR3=0000000000000000 
DR6=00000000ffff0ff0 DR7=0000000000000400
EFER=0000000000000100
Code=<??> ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ?? ??
```

Reply:
Yeah, that seems strange...  I would have expected the encryption support to be the big user of stack space, but it looks like the encryption succeeds. Maybe it's the decryption of a larger blob that is causing the issue. It would be nice to understand the openssl stack usage.

Set the stack to the number of pages you need and keep the AP stack size in sync (since an attestation request could come from an AP, too).